packages/kbn-esql-validation-autocomplete/src/autocomplete/autocomplete.ts

```diff
@@ -349,8 +349,8 @@ function getSourcesRetriever(resourceRetriever?: ESQLCallbacks) {
     return buildSourcesDefinitions(
       list
         .filter(({ hidden }) => !hidden)
-        .map(({ name, dataStreams, title }) => {
-          return { name, isIntegration: Boolean(dataStreams && dataStreams.length), title };
+        .map(({ name, dataStreams, title, type }) => {
+          return { name, isIntegration: Boolean(dataStreams && dataStreams.length), title, type };
```

Comment:
Does it make sense to merge the `isIntegration` and `type` properties? For example, integrations would be `type:Integration`.

Reply:
It comes translated so it won't be a good way to test it, right?

Reply:
A fair point. Didn't connect the `type` with translation